tests/run-tests

```diff
@@ -7,17 +7,37 @@
 #
 set -e
 
+
```

Comment:
Moving this function up here is a bit ugly. The full normalised path is only required because cram is running in a temp dir, so can we delay normalising the path until just before we call cram? It looks like `package_dir` is exported again later so we can just normalise it as part of that, e.g.:

```
export PACKAGE_DIR="$(_normalise_path "$package_dir")"
```